include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -174,4 +174,10 @@ def BlackBoxReader : Pass<"firrtl-blackbox-reader", "CircuitOp"> {
   let dependentDialects = ["sv::SVDialect"];
 }
 
+def GrandCentralTaps : Pass<"sifive-gct-taps", "firrtl::CircuitOp"> {
```

Comment:
The trouble is that the pass converts a bunch of `firrtl.extmodule`s into `firrtl.module`s, but then also goes around to change some of the symbol refs to that module. There can be weird circumstances where one extmodule expands to multiple regular modules depending on where they are instantiated (because the body of the GCT data taps depends on its instantiation location in weird ways). Not sure how well that plays with multithreading :thinking:

Reply:
yeah agreed, got it, thx